lib/standards.h

```diff
@@ -79,6 +85,12 @@ struct Standards {
         }
         return false;
     }
+    const std::string getCPP(void) const {
+        if (cpp == CPP03) return "c++03";
+        else if (cpp == CPP11) return "c++11";
+        else if (cpp == CPP14) return "c++14";
```

Comment:
I would prefer a switch here without "default" so gcc will warn if there is a missing enum constant.

Reply:
Some were not handled, so that seems like a good idea.  Fixed.